src/components/Details/Details.tsx

```diff
@@ -1,7 +1,66 @@
-import React from 'react';
+import React from "react";
+import "../Details/Details.css"
 
-interface DetailsProps {}
+interface MarketProps {
+  markets: {
+    id: number;
+    distanceFromZip: number;
+    marketName: string
+  }[],
+  marketDetails: {
+    street: string,
+    city: string,
+    state: string,
+    zip: string,
+    schedule: {
+      dayOfWeek: string,
+      time: string,
+      season: string
+    }[],
+    products: string[],
+    mapsLink: string
+  }[],
+  id: string;
+}
+
+export const Details: React.FC<MarketProps> = ({ id, markets, marketDetails }) => {
+
+  const marketMatch = markets.filter(market => market.id === Number(id));
+  const detailsMatch = marketDetails.filter(market => { 
+    const match = market.mapsLink.split('%22')[1].includes(marketMatch[0].marketName.split(' ')[0]);
```

Comment:
Should we wait on merging this until Alex/someone can get the id out of our utils function?